### DISCUSSION TITLE
Developer Prompts

### DISCUSSION THREAD

**Discussion by jacobdaddario** [joemasilotti/railsdevs.com#412] [Ideas]
Right now, the developer page mostly consists of the Bio and Hero text. I propose, the developer page could be fleshed out by adding optional discussion prompts. These are a series of questions/discussion topics that developers can select and then write responses too.

Ideally I think there could be a short character limit, and these responses could be used to fill out a developer card view. These card views could be used to improve the index/search view for when companies are searching or developers.

**Comment by joemasilotti** [comment]
I freaking _love_ this idea Jacob! What kind of prompts did you have in mind?

**Reply by jacobdaddario** [reply]
I think that we could brainstorm a couple, and once we settle on what we want to do, I can go ahead and do a first pass implementation. Here's some of my ideas:

1. What's is a piece of your own open source work that you're proud of?
2. What domains or problems interest you and why?
3. What do more tech managers need to hear?
4. Prototype first and refine or design first implement second?
5. What makes for a high-quality pull request?

These are just a couple of ideas off the top. I'd love to field a few more suggestions. Twitter might be a good resource for this.

**Reply by joemasilotti** [reply]
Off to a great start, as always! Some more suggestions:

1. Why do _you_ test your code?
2. What brought you to Rails initially? What makes you stay?
3. What's your favorite under appreciated or often overlooked part of Rails?
4. What other languages and frameworks do you enjoy to work with?
